build.sbt

```diff
@@ -11,7 +11,12 @@ import Sxr.sxr
 // but can be shared across the multi projects.
 def buildLevelSettings: Seq[Setting[_]] = Seq(
   organization in ThisBuild := "org.scala-sbt",
-  version in ThisBuild := "0.13.9-SNAPSHOT"
+  version in ThisBuild := "0.13.9-SNAPSHOT",
+  // bintrayOrganization in ThisBuild := None,
+  // bintrayRepository in ThisBuild := "test-test-test",
+  bintrayOrganization in ThisBuild := Some("sbt"),
```

Comment:
this is fine, but FYI, we'll have to mirror this on the typesafe org so the URLs match the old one.